src/app.js

```diff
@@ -54,6 +54,12 @@ module.exports = function (badgerService) {
     state: 'none'
   });
 
+  app.use(function (req, res, next) {
+    res.header('Access-Control-Allow-Origin', '*');
+    res.header('Access-Control-Allow-Headers', 'Origin, X-Requested-With, Content-Type, Accept');
```

Comment:
The only thing to consider is whether there are ever going to be additional headers that may be sent from the client. Things like `Cache-Control` or other browser headers that may be injected?